### ISSUE TITLE
Scrolling became intermittent in 1718 build

### ISSUE THREAD

**Issue by GoogleCodeExporter** [JustLoy77/ebookdroid#329]
```
What steps will reproduce the problem?
1. Update EBookDroid to 1718 build
2. Try to scroll pages quickly

What is the expected output?
I can't scrol pages quickly, only two pages for one long tap.

What do you see instead?
I want to see continuous smooth scrolling with support of speed-up at often 
long taps and speed-down at single long tap.

What version of the product are you using?
Ebookdroid 1.5.3, build 1718

On what operating system?
Android 3.2

What device are you using?
Galaxy Tab 7.7 P6800

```

Original issue reported on code.google.com by `skurydi...@gmail.com` on 10 Oct 2012 at 2:57

**Comment by GoogleCodeExporter** [comment]
```
Besides, if to choose horizontal scrolling, this bug disappears. But each page 
needs to be centered manually after horizontal scrolling.
```

Original comment by `skurydi...@gmail.com` on 11 Oct 2012 at 9:03

**Comment by GoogleCodeExporter** [comment]
```
So, bug is present in vertical scroll and not present in horizontal? Hmmm... 
Very strange...
```

Original comment by `Andrei.K...@gmail.com` on 11 Oct 2012 at 9:45

**Comment by GoogleCodeExporter** [comment]
```
Yes. Maybe it depends on lines, cached previously...
```

Original comment by `skurydi...@gmail.com` on 11 Oct 2012 at 9:59

**Comment by GoogleCodeExporter** [comment]
```
The options of scrolling are overloaded. For example, in Aldiko reader there 
are no scrolling options and scrolling is center-oriented without any freezes 
or page cuttings.
```

Original comment by `skurydi...@gmail.com` on 12 Oct 2012 at 7:50

**Comment by GoogleCodeExporter** [comment]
```
Fixed (I hope) in 1.5.5
```

Original comment by `Andrei.K...@gmail.com` on 17 Oct 2012 at 8:58
- Changed state: **Fixed**

**Comment by GoogleCodeExporter** [comment]
```
I mean, it would be nice to see only two types of scrolling in options:
a) page-oriented and center-oriented (as in Aldiko)
b) line-oriented (as in current version) - for very small devices with zooms 
<1.0.

PayPal donation is wanted via appropriate button on first web-page.
See an exaple at:
http://code.google.com/p/rt-n56u/

Great thanks.
```

Original comment by `skurydi...@gmail.com` on 17 Oct 2012 at 9:07
